packages/cozy-client/types/context.d.ts

```diff
@@ -1,6 +1,6 @@
 export const clientContext: import("react").Context<{
```

Comment:
those types are created automatically, right? why are they versioned instead of built before deployment? this question is not strictly related to the current PR obviously and doesn't block it

Reply:
I'm not sure why this has been done.